conf/ocsci/aws_ipi_workload_3_nodes.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# This config will provide you the environment with:
+#  * 3 master nodes
+#  * 6 worker nodes where:
+#    * 3 worker nodes used for OCS
+#    * 3 worker nodes used for workload
+#
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  master_replicas: 3
+  worker_replicas: 6
+  ocs_operator_nodes_to_label: 3
+  ocs_operator_nodes_to_tain: 3
```

Comment:
taint? or tain?

Reply:
This was typo in previous change. So it expected  ocs_operator_nodes_to_tain, but I fixed it next commit